include/cantera/thermo/ThermoFactory.h

```diff
@@ -107,6 +107,24 @@ inline ThermoPhase* newThermoPhase(const std::string& model)
     return ThermoFactory::factory()->create(model);
 }
 
+//! Create a new ThermoPhase instance.
+/*!
+ * @param model   String to look up the model against
+ * @returns a shared pointer to a new ThermoPhase instance matching the model string.
+ *   Throws an exception UnknownThermoPhaseModel if the string wasn't matched; or a
+ *   CanteraError if something else went wrong.
```

Comment:
This isn't actually true -- the `UnknownThermoPhaseModel` exception class is never used, and already deprecated. I guess this bit of erroneous documentation from `newThermoPhase` ought to be corrected.

Reply:
copypasta :joy: